RetroBar/Languages/עברית.xaml

```diff
@@ -0,0 +1,65 @@
+﻿<ResourceDictionary xmlns="http://schemas.microsoft.com/winfx/2006/xaml/presentation"
+                    xmlns:x="http://schemas.microsoft.com/winfx/2006/xaml"
+                    xmlns:s="clr-namespace:System;assembly=mscorlib">
+
+    <s:String x:Key="retrobar_properties">מאפייני RetroBar</s:String>
+    <s:String x:Key="taskbar_options">הגדרות שורת המשימות</s:String>
+    <BitmapImage x:Key="properties_95" UriSource="../Resources/he/properties.png" />
+    <BitmapImage x:Key="properties_95_noclock" UriSource="../Resources/he/properties_noclock.png" />
+    <s:String x:Key="autostart">התחל אוטומטית בכ_ניסה</s:String>
+    <s:String x:Key="language_text">שפ_ה:</s:String>
+    <s:String x:Key="language_tip">בחר את השפה שלך.</s:String>
+    <s:String x:Key="theme_text">_עיצוב:</s:String>
+    <s:String x:Key="theme_tip">התקן ערכות נושא בתיקיה "Themes".</s:String>
+    <s:String x:Key="location_text">מי_קום:</s:String>
+    <s:String x:Key="location_tip">שנה את מיקום שורת המשימות.</s:String>
+    <x:Array x:Key="location_values" Type="s:String">
+        <s:String>שמאל</s:String>
+        <s:String>סביבון</s:String>
+        <s:String>ימין</s:String>
+        <s:String>ישבן</s:String>
+    </x:Array>
+    <s:String x:Key="allow_font_smoothing">_אפשר החלקת גופנים</s:String>
+    <s:String x:Key="collapse_tray_icons">סמלי אזור הודעה מכווץ</s:String>
+    <s:String x:Key="show_clock">הצג את שעון המערכת</s:String>
+    <s:String x:Key="show_multi_mon">Show on _multiple displays</s:String>
```

Comment:
@novatiberium, there are a couple of lines left to be translated for the release, could you update the translation?

Reply:
@novatiberium, if you still want to update the translation, just open another PR.